assets/src/components/notificationCard.tsx

```diff
@@ -182,20 +190,48 @@ const description = (
     }
 
     case NotificationType.Detour: {
-      return (
-        <>
-          <div className="d-flex flex-row gap-2">
-            <RoutePill routeName={notification.content.route} />
-            <div>
-              <div className="fw-semibold">{notification.content.headsign}</div>
-              <div className="fw-normal text-body-secondary">
-                From {notification.content.origin.split(" - ")[0]}
+      switch (notification.content.status) {
```

Comment:
The returns for both the `Activated` and `Deactivated` cases are identical. Is this switch needed?

Reply:
Oh y'know, that's true, I'll remove the switch